### PR TITLE
Single Video Page

### PR DESCRIPTION
**Pull request by yogesh-**
Completed single video page.

image

### CODE REVIEW

**Review by agrawalshanu708 — APPROVED** [review]
file structure is good
UI looks good, but you can improve it more
u can merge

File: src/pages/singleVideoPage/singelVideoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import MenuIcon from "@mui/icons-material/Menu";
+import { Search } from "../../components/searchBox/searchbox";
```
Comment (pariyar07) [inline src/pages/singleVideoPage/singelVideoPage.jsx:2]: Use absolute imports here makes it clean and less confusing.

File: src/routing.jsx [hunk 2]
```diff
@@ -19,6 +20,7 @@ export const Routing = () => {
           <Route path="/signup" element={<SignUp />} />
           <Route path="/likes" element={<Likes />} />
           <Route path="/watchlater" element={<WatchLater />} />
+          <Route path="/playvideo/:videoId" element={<SingleVideoPage />} />
         </Route>
```
Comment (HarshnaNHaswani) [inline src/routing.jsx:24]: can use useRoutes() instead

**Review by HarshnaNHaswani — APPROVED** [review]
Good to go
- clean code
- good file structure
- can increase padding/margin for the side menu

File: src/routing.jsx (hunk 2 above)
Comment (yogesh-) [reply src/routing.jsx:24]: okay will explore.

File: src/pages/singleVideoPage/singelVideoPage.jsx (hunk 1 above)
Comment (yogesh-) [reply src/pages/singleVideoPage/singelVideoPage.jsx:2]: ok, will optimize

**Review by pariyar07 — APPROVED** [review]
Good naming of components. Good flow of data. Looks good to merge.

**Pull request merged**